lib/ex_unit/lib/ex_unit/on_exit_handler.ex

```diff
@@ -2,20 +2,40 @@ defmodule ExUnit.OnExitHandler do
   @moduledoc false
 
   @name __MODULE__
-  @supervisor 2
-  @on_exit 3
   @ets_opts [:public, :named_table, read_concurrency: true, write_concurrency: true]
 
+  # ETS column numbers
+  @pid 1
+  @supervisor 3
+  @on_exit 4
+
   use Agent
 
   @spec start_link(keyword()) :: {:ok, pid}
   def start_link(_opts) do
     Agent.start_link(fn -> :ets.new(@name, @ets_opts) end, name: @name)
   end
 
-  @spec register(pid) :: :ok
-  def register(pid) when is_pid(pid) do
-    :ets.insert(@name, {pid, nil, []})
+  @spec register(pid, term) :: :ok
```

Comment:
Is the `manager` also here `{pid, pid}`? (same for `get_registered_pids /1`)

Reply:
yes. I guess we could enforce that in the spec

Reply:
I have added it as a type